cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/prometheus/DefaultPrometheusQuerySupplier.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.monitor.sampling.prometheus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.linkedin.kafka.cruisecontrol.metricsreporter.metric.RawMetricType;
+
+import static com.linkedin.kafka.cruisecontrol.metricsreporter.metric.RawMetricType.*;
+
+/**
+ * Default prometheus query supplier. This can be used if the Prometheus JMX and node exporters on the
+ * Kafka brokers are running with standard vanilla configuration, and do not have any special rules for
+ * transformation of metric names.
+ *
+ * See {@link PrometheusQuerySupplier}
+ */
+public class DefaultPrometheusQuerySupplier implements PrometheusQuerySupplier {
+    private static final Map<RawMetricType, String> TYPE_TO_QUERY = new HashMap<>();
+    static {
+        // broker metrics
+        TYPE_TO_QUERY.put(BROKER_CPU_UTIL,
+            "1 - avg by (instance) (irate(node_cpu_seconds_total{mode=\"idle\"}[1m]))");
+        TYPE_TO_QUERY.put(ALL_TOPIC_BYTES_IN,
+            "kafka_server_BrokerTopicMetrics_OneMinuteRate{name=\"BytesInPerSec\",topic=\"\"}");
```

Comment:
Hi @wyuka, 

I'm sorry for digging into this old stuff, but I'm having a problem understanding this particular query and would really appreciate it if you could help. 

Here why would we want to send a query with the label `topic=""`? Based on my experience this query always returns an empty result, but if we just omit the topic label then it works fine. Also, since this is broker type metric, wouldn't it be better to prepend it with `sum by (instance)`? So, my question is shouldn't this query look something like `kafka_server_BrokerTopicMetrics_OneMinuteRate{name=\"BytesInPerSec\"}`, or even `sum by (instance) (kafka_server_BrokerTopicMetrics_OneMinuteRate{name=\"BytesInPerSec\"})`?

I guess the same question to all other queries that have label `topic=""`.

Thanks in advance!